src/ray/core_worker/reference_count.h

```diff
@@ -106,18 +143,79 @@ class ReferenceCounter {
   std::unordered_map<ObjectID, std::pair<size_t, size_t>> GetAllReferenceCounts() const
       LOCKS_EXCLUDED(mutex_);
 
+  /// Populate a table with ObjectIDs that we were or are still borrowing.
+  /// This should be called when a task returns, and the argument should be any
+  /// IDs that were passed by reference in the task spec or that were
+  /// serialized in inlined arguments.
+  ///
+  /// See GetAndClearLocalBorrowersInternal for the spec of the returned table
+  /// and how this mutates the local reference count.
+  ///
+  /// \param[in] borrowed_ids The object IDs that we or another worker were or
+  /// are still borrowing. These are the IDs that were given to us via task
+  /// submission and includes: (1) any IDs that were passed by reference in the
+  /// task spec, and (2) any IDs that were serialized in the task's inlined
+  /// arguments.
+  /// \param[out] proto The protobuf table to populate with the borrowed
+  /// references.
+  void GetAndClearLocalBorrowers(const std::vector<ObjectID> &borrowed_ids,
+                                 ReferenceTableProto *proto) LOCKS_EXCLUDED(mutex_);
+
+  /// Wrap an ObjectID(s) inside another object ID.
```

Comment:
```suggestion
  /// Wrap ObjectIDs inside another object ID.
```

Reply:
What does this actually do? The name of the function nor the description here really make it clear IMO. `WrapObjectIDs` sounds more like we're serializing IDs within one another.